modules/raft/src/integrationTest/java/org/apache/ignite/raft/jraft/core/ItNodeTest.java

```diff
@@ -2710,28 +2717,33 @@ public void testAppendEntriesWhenFollowerIsInErrorState() throws Exception {
     }
 
     @Test
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-21792")
+    @Timeout(value = 25, unit = TimeUnit.SECONDS)
     public void testFollowerStartStopFollowing() throws Exception {
         // start five nodes
         List<TestPeer> peers = TestUtils.generatePeers(testInfo, 5);
 
-        cluster = new TestCluster("unitest", dataPath, peers, ELECTION_TIMEOUT_MILLIS, testInfo);
+        cluster = new TestCluster("unittest", dataPath, peers, ELECTION_TIMEOUT_MILLIS, testInfo);
 
         for (TestPeer peer : peers)
             assertTrue(cluster.start(peer));
         Node firstLeader = cluster.waitAndGetLeader();
         assertNotNull(firstLeader);
         cluster.ensureLeader(firstLeader);
 
+        //we don't want any timeouts after the first leader election for this test
+        for (Node nodes : cluster.getNodes()) {
+            nodes.resetElectionTimeoutMs(100_000);
```

Comment:
Unfortunately, this do not prevent election, as you've expected.

As you can see in the test scenario further, we stop current leader

```
        // stop leader and elect new one
        PeerId fstLeaderAddr = firstLeader.getNodeId().getPeerId();
        assertTrue(cluster.stop(fstLeaderAddr));
```
And new leader is elected in a short time. This is because of the `TimeoutNowRequest` optimisation. 

To sum up, I don't think that this idea to reset timeout to some big value could bring us any profit.

Reply:
Hi @alievmirza thanks for your review!. I don't quite understand your point. From my point of view, what happened was that the flakiness didn't occur when the leader was stopped, but only when running into the actual election timeout. The purpose of this test is to take advantage of the fact that when a leader is stopped, a new round of election is triggered. 

Maybe the comment is confusing because I didn't want to call a "leader stop" an election timeout (even though under the hood this may use TimeoutNowRequest). Or are we talking about a completely different timeout mechanism? I didn't came across this TimeoutNowRequest while searching for the flakiness.